src/nvim/ops.c

```diff
@@ -5519,8 +5511,9 @@ static bool get_clipboard(int name, yankreg_T **target, bool quiet)
     reg->y_type = kMTUnknown;
   }
 
-  reg->y_array = xcalloc(lines->lv_len, sizeof(uint8_t *));
-  reg->y_size = lines->lv_len;
+  assert(lines->lv_len >= 0);
```

Comment:
I think it safe to assume the length of a list is nonnegative. (we could have asserts in the functions for manipulating lists, but not in every case a list is just being used).